packages/account/src/Sections/Security/ApiToken/__tests__/api-token.spec.tsx

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
 import { FormikValues } from 'formik';
-import { act, render, screen, waitFor } from '@testing-library/react';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
please delete this fireEvent if not used anywhere, if used, please check if it can be userEvent. 🖱️